R/coord-.R

```diff
@@ -110,7 +110,9 @@ Coord <- ggproto("Coord",
     scale_position <- lapply(panel_params[aesthetics], `[[`, "position")
     guide_position <- lapply(guide_params, `[[`, "position")
     guide_position[!is_sec] <- Map(
-      `%|W|%`, guide_position[!is_sec], scale_position[!is_sec]
+      function(guide, scale) guide %|W|% scale,
+      guide = guide_position[!is_sec],
+      scale = scale_position[!is_sec]
```

Comment:
Nothing wrong with the prior version, so which one is up to you. My comment was only towards setter functions